src/services/rfqm_service.ts

```diff
@@ -0,0 +1,130 @@
+import { AssetSwapperContractAddresses, MarketOperation, ProtocolFeeUtils, QuoteRequestor } from '@0x/asset-swapper';
+import { BigNumber } from '@0x/utils';
+
+import { META_TX_WORKER_REGISTRY, RFQT_REQUEST_MAX_RESPONSE_MS } from '../config';
+import { NULL_ADDRESS, ONE_MINUTE_MS } from '../constants';
+
+export interface FetchIndicativeQuoteParams {
```

Comment:
we'll need a fee param potentially, worth adding now?

Reply:
Chat w/ @PirosB3 - won't add the fee param now, but will keep the `gas` value as the fee is likely to be dependent on the gas value

Reply:
Hey @alexkroeger just wanted to follow up here. We are going to get this code merged first and then quickly follow up with the fee upgrade. I'm biasing towards getting some code in since we have plenty of pending code.